src/components/control/select.jsx

```diff
@@ -112,7 +112,7 @@ const Select = forwardRef(function Select(props, ref) {
 
   useImperativeHandle(selfRef, () => ({
     get dirty() {
-      return props.value === undefined ? value !== null : props.value !== value;
+      return props.value !== this.value;
```

Comment:
Just curious, how does this work or what is going on here?

Reply:
`this.value` is the result from the `get value()` below. So it's basically what the component thinks it's current state is, like what you would send to the server, i.e. the non-persisted value

`props.value` is what is passed down from "somewhere" -- this somewhere is usually an XHR that is from the server, i.e. the persisted value. 

So we know the component is "dirty" when it does not match up!